src/app/containers/RadioSchedule/Canonical/useData.jsx

```diff
@@ -0,0 +1,31 @@
+import { useState, useEffect } from 'react';
+import webLogger from '#lib/logger.web';
+
+const logger = webLogger();
+
+const useData = (endpoint, initialData) => {
+  const [data, setData] = useState(initialData);
+
+  useEffect(() => {
+    const handleResponse = async response => {
+      if (!response.ok) {
+        throw Error(response.statusText);
+      }
+      const json = await response.json();
+      setData(json);
+    };
+
+    const fetchData = pathname =>
+      fetch(pathname, { mode: 'no-cors' })
```

Comment:
Have left 'no-cors' in, as this was in the useEffect in radio schedules. Just want to double check why we're using it!